x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.test.ts

```diff
@@ -755,6 +755,108 @@ describe('EPM template', () => {
     expect(mappings).toEqual(objectFieldWithPropertyReversedMapping);
   });
 
+  it('tests processing object field with subobjects set to false (case A)', () => {
+    const objectFieldWithPropertyReversedLiteralYml = `
+- name: a.labels
+  type: object
+  subobjects: false
```

Comment:
I think that this mapping is not allowed in the package spec. It would not define a mapping for the sub fields of `a.labels`.

Reply:
Yep, I agree. 

I kept this case around if we found a compelling use case to resurrect it. There isn't one.

So, since it is not allowed in package-spec v3, I will drop this mapping from the PR.